### ISSUE TITLE
Review some sin texts

### ISSUE THREAD

**Issue by JohnRDOrazio** [kas-catholic/confessit-web#61]
I would suggest reviewing some of the texts for certain "sins". I figure it's easier to track progress and discuss here than on Slack, where things can get lost in a scattered conversation. We can discuss with comments on this issue, and then as a consensus is drawn, the following table can be updated with the results:

| Sin ID | Current | Suggested | State |
|:-------:|:--------:|:------------:|:-------|
| `sins.17.text` | [Did I fail to teach my children about their human sexuality?](https://translate.johnromanodorazio.com/translate/confessit-web/public-locales/en/?checksum=54af441074ab2f62&q=sins&sort_by=-priority%2Cposition) | **Did I fail to teach my children about the sanctity of marriage and the responsible exercise of human sexuality according to God's plan?** | <ul><li>- [x] in progress</li><li>- [ ] implemented</li><li>- [ ] will not change</li></ul>  |
| `sins.32.text` | [Did I give in to overly passionate kissing for pleasure?](https://translate.johnromanodorazio.com/translate/confessit-web/public-locales/en/?checksum=ba0cc91d60827159&q=sins&sort_by=-priority%2Cposition) | **Did I fail to avoid the near occasions of sin in relating with my fiancée/fiancé?** | <ul><li>- [x] in progress</li><li>- [ ] implemented</li><li>- [ ] will not change</li></ul> |
| `sins.34.text` | [Did I pirate movies, music, or software?](https://translate.johnromanodorazio.com/translate/confessit-web/public-locales/en/?checksum=c87bc9ec5aac0b18&q=sins&sort_by=-priority%2Cposition) | | <ul><li>- [x] in progress</li><li>- [ ] implemented</li><li>- [ ] will not change</li></ul> | 
| `sins.44.text` | [Was I jealous of someone else's wife or fiancée?](https://translate.johnromanodorazio.com/translate/confessit-web/public-locales/en/?checksum=de0c66b22af32a45&q=sins&sort_by=-priority%2Cposition) | should mention husbands and fiancés 😄 | <ul><li>- [ ] in progress</li><li>- [x] implemented</li><li>- [ ] will not change</li></ul> | 
| `sins.46.text` | [Did I vote for a candidate or policy that does not uphold my Catholic values?](https://translate.johnromanodorazio.com/translate/confessit-web/public-locales/en/?checksum=a844d81535f1a64a&q=sins&sort_by=-priority%2Cposition) | Did I fail to uphold Catholic values when fulfilling my civic duties? Did I uphold policies contrary to Catholic values? | <ul><li>- [x] in progress</li><li>- [ ] implemented</li><li>- [ ] will not change</li></ul> |

**Comment by JohnRDOrazio** [comment]
`sins.44.text` was implemented in PR #51

**Comment by kas-catholic** [comment]
:+1: on 17!

**Comment by kas-catholic** [comment]
32 was originally intended not only for fiancée/fiancé but also boyfriend/girlfriend (or anyone other than a husband/wife, for that matter). Maybe instead of saying "fiancée/fiancé" we just say "before marriage"? Is that what you were getting at? Thoughts?

**Comment by kas-catholic** [comment]
34 - How about, "Did I steal intellectual property (movies, music, software, etc.)?"

* I think "wrongfully" is implied if it's a sin? How do you not-wrongfully steal?
* I lean toward including the movies/music example one way or another because I don't think it always occurs to people that downloading music could be a sin when they're examining their conscience. That might seem obvious to some people but IMO this should be targeted at a wide audience. 

Thoughts?

**Comment by kas-catholic** [comment]
46 - How about, "Did I fail to uphold Catholic values when voting? Did I uphold policies contrary to Catholic values?" or simlar? ("Fulfilling civic duties" sounds odd to me vs simply "voting").

**Comment by JohnRDOrazio** [comment]
> 32 was originally intended not only for fiancée/fiancé but also boyfriend/girlfriend (or anyone other than a husband/wife, for that matter). Maybe instead of saying "fiancée/fiancé" we just say "before marriage"? Is that what you were getting at? Thoughts?

Makes sense. Perhaps, **Did I fail to avoid the near occasions of sin in dealing with a fiancée/fiancé or girlfriend/boyfriend?**

I'm not sure how I would put it if trying to use the expression "before marriage", or where it would fit in. The "near occasions of sin" in this context are in reference to the relationship with another person. So a mention of the relationship with another person gives meaning to this point of examination. If we say "in the relationship with my fiancée/fiancé or boyfriend/girlfriend", it's obvious we're talking about "before marriage". Instead saying something like **Did I avoid the near occasions of sin before marriage** seems to me a little bit more abstract, not mentioning the context of a relationship.

**Comment by JohnRDOrazio** [comment]
> 34 - How about, "Did I steal intellectual property (movies, music, software, etc.)?"
> 
> * I think "wrongfully" is implied if it's a sin? How do you not-wrongfully steal?
> * I lean toward including the movies/music example one way or another because I don't think it always occurs to people that downloading music could be a sin when they're examining their conscience. That might seem obvious to some people but IMO this should be targeted at a wide audience.
> 
> Thoughts?

My thoughts are as I expressed on Slack, that there are a lot of grey areas here. It's not all black and white, which is why I specified `wrongfully`. There can be certain circumstances where it's not actually stealing from the rightful owners: take for example Spotify. Nowadays it's official. When it started, it wasn't official, and it had to go through a lot of legal dealings. In the end, they were able to make this agreement with the Record Labels, that there would be a free version with Ads and limited access to songs, and a subscription model to have full access to songs without Ads.

It's not always clear who the rightful owners of intellectual property are, for example when it comes to music: the songwriter? the singer? the guitar player? the record label that publishes the song? Oftentimes the record labels would take advantage of the situation, and the actual "creator" (songwriter / singer) wouldn't get their due, whereas the record labels would become rich off of someone elses creation. Again, this isn't all black and white either: Record Labels often act as a coach, in bringing a song to perfection, so they do at times have merit in the creative process.

It's easy (and lazy) to see situations in black and white. But spiritual reality is more complex. Even when it comes to using software illegally, using a "cracked" version of a program, there is a big difference between gaining access to software for educational purposes, and taking instead advantage at a professional level when you're trying to make money and not pay the rightful owners of the software you use to make your money.

Microsoft has even come to terms with this: until a few years ago, the tools for writing C / C++ code and compiling it (Visual Studio) cost about $900 a year. Could you ever imagine a student who's trying to learn C++, paying $900 a year just to test it out in the learning process? That's where Linux had it's strength: you could write and compile code for free, depending on the generosity of a community interested more in world development that in making a profit.

Finally Microsoft started creating "community" editions at a discounted price, even though these "community" editions had so many limitations that you were very restricted in what you could write and compile. They also started creating "educational" packages, allowing universities and schools greater access to tools for learning purposes.

And low and behold, Microsoft has now embraced Linux, embedding it in it's own operating system and creating interoperability. And it bought Github! Opening up to the open source community.

Would all of this have ever happened if nobody had ever "cracked" and used their software illegally? They realized they had to come to terms with the good of the community.

So these situations are very nuanced, not always black and white. The real root of the moral dilemma, is whether or not I am trying to have a personal economical advantage, creating damage to someone else in the process, who has a right to a just recompense for their labor.

I agree that we can specify what **intellectual property** refers to, leaving an open list, so I would suggest:

`Did I steal intellectual property (movies, music, software, etc.) to my own advantage, damaging the rightful owners?`

**Comment by JohnRDOrazio** [comment]
> 46 - How about, "Did I fail to uphold Catholic values when voting? Did I uphold policies contrary to Catholic values?" or simlar? ("Fulfilling civic duties" sounds odd to me vs simply "voting").

This works for me.

**Comment by kas-catholic** [comment]
* **32:** I understand your point about relationship. This is just me nitpicking at this point, but I think both "dealing with" and "relating with" sound odd, how about, "Did I fail to avoid the near occasions of sin in my relationship with a fiancée/fiancé or girlfriend/boyfriend?" I'm ultimately fine with whatever you decide here.
* **34:** I understand the point you're making. With that context, I think I like your original, but with examples: "Did I wrongfully steal intellectual property (movies, music, software, etc.)?"
